<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Система массового обслуживания</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>qt_ico.png</normaloff>qt_ico.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>370</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="1">
      <widget class="QComboBox" name="DistrTypeBox">
       <property name="currentText">
        <string>Экспоненциальное</string>
       </property>
       <item>
        <property name="text">
         <string>Экспоненциальное</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Линейное</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="TimePopMinEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ServerCountEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество серверов: </string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Интенсивность потока заявок:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="TimeExecMaxEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="RunsNumEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Мин. время поступления(лин):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Время работы системы: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Размер буфера:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="TimeEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Количество прогонов:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="BufferCountEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Макс. время поступления(лин):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Мин. время обработки(лин):</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Распределение</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="TimePopMaxEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="TimeExecMinEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Макс. время обработки(лин):</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Интенсивность потока обрабоки:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="LambdaEdit"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="MuEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Обработать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
